lib/kredis/types/expirable.rb

```diff
@@ -0,0 +1,42 @@
+module Kredis::Types::Expirable
+  extend ActiveSupport::Concern
+
+  included do
+    proxying :expire, :expireat, :ttl
+
+    attr_accessor :expires_in, :expires_at
+
+    def expire_in(seconds)
+      expire seconds.to_i
+    end
+
+    def expire_at(datetime)
+      expireat datetime.to_i
+    end
+
+    def refresh_expiration
+      expire_in(@expires_in) if expires_in
+      expire_at(@expires_at) if expires_at && !expires_in
+    end
```

Comment:
You may consider adding a `clear_expiration` that does `expire 0` or `persist` Redis commands...but that could be a separate addition.